roles/sap_anydb_install_oracle/tasks/oracledb_install_pre.yml

```diff
@@ -0,0 +1,133 @@
+---
+
+# See further:
+## oracle-database-preinstall-12cr1.rpm
+## oracle-database-preinstall-12cr2.rpm
+## oracle-database-preinstall-19c.rpm >> https://yum.oracle.com/repo/OracleLinux/OL8/appstream/x86_64/getPackage/oracle-database-preinstall-19c-1.0-2.el8.x86_64.rpm
+- name: Install C Shell (CSH), Korn Shell (KSH) and other dependencies for Oracle DB
+  ansible.builtin.package:
+    name:
+    - csh
+    - ksh
+    - mksh
+    - bc
+    - bind-utils
+    - gcc
+    - gcc-c++
+    - glibc-devel
+    - libcap
+    - libaio
+    - libaio-devel
+    - libgcc
+    - libstdc++
+    - libstdc++-devel
+    - make
+    - psmisc
+    - rdma-core-devel
+    - smartmontools
+    - sysstat
+    - unzip
+    - xorg-x11-utils
+    - xorg-x11-xauth
+    - unixODBC
+    - unixODBC-devel
+    - libnsl
+    state: present
+  notify: __oracledb_prep_reboot_handler
```

Comment:
As per our [style guide](https://sap-linuxlab.github.io/contribute/contribute_04_dev_guidelines/), rename this variable to __sap_anydb_install_oracle_reboot_handler here and in handlers/main.yml